src/Io/KeyboardInputHandler.cpp

```diff
@@ -79,12 +82,16 @@ void Io::KeyboardInputHandler::GenerateGameplayActions() {
         bool isTriggered = false;
         PlatformKey key = actionMapping->GetKey(action);
         PlatformKey gamepadkey = actionMapping->GetGamepadKey(action);
-        if (GetToggleType(action) == KeyToggleType::TOGGLE_OneTimePress) {
+
+        switch (GetToggleType(action)) {
+        default: assert(false); [[fallthrough]];
```

Comment:
Should this not be the last case in the switch? Outside of debug mode this will cause the program to run `TOGGLE_OneTimePress`, is this intented behaviour?

Reply:
This is mostly about making (some) compilers happy. `TOGGLE_OneTimePress` is a sane default, but the intention is that `default` will never be triggered.